documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -18,6 +18,10 @@ description = "Auxiliary images are an alternative approach for supplying a doma
      - [Configuration example 1: Basic configuration](#configuration-example-1-basic-configuration)
      - [Configuration example 2: Source locations](#configuration-example-2-source-locations)
      - [Configuration example 3: Multiple images](#configuration-example-3-multiple-images)
+ - [Automated upgrade of the weblogic.oracle/v8 schema auxiliary images configuration](#automated-upgrade-of-the-weblogicoraclev8-schema-auxiliary-images-configuration)
+   - [Sample weblogic.oracle/v8 schema auxiliary image configuration](#sample-weblogicoraclev8-schema-auxiliary-image-configuration)
```

Comment:
Title should have back-ticks around the weblogic.oracle/v8 keyword

Reply:
Done.